zk/stages/stage_sequence_execute.go

```diff
@@ -146,7 +174,7 @@ func SpawnSequencingStage(
 		limboHeaderTimestamp = math.MaxUint64
 	}
 
-	if l1Recovery {
+	if l1Recovery && isLastBatchFinished {
```

Comment:
will this cause the recovery to stop after the first batch has been processed during the recovery?

Reply:
I expect this to finish last started batch before going into recovery. Now that I think of it there is a case where te batch can't be finished and hangs, so maybe we should do this some other way.

Reply:
Thinking about it again this morning, I think we can just remove the `isLastBatchFinished` part.  L1 recovery shouldn't get into a position where we can't finish the batch.

Reply:
Okay, going to remove it